hal/armv7m/imxrt/10xx/console.c

```diff
@@ -17,17 +17,21 @@
 #define UART_CONSOLE 1
 #endif
 
-#ifndef CONSOLE_BAUDRATE
-#define CONSOLE_BAUDRATE 115200
-#endif
-
 #include "cpu.h"
 #include "console.h"
 #include "imxrt10xx.h"
 #include "../../../../include/errno.h"
 #include "../../../../include/arch/imxrt.h"
 
-#define UART_CLK 80000000
+#define CONCAT(a, b)     a##b
+#define UART_PIN(n, pin) CONCAT(UART, n##_##pin)
+#define CONSOLE_BAUD(n)  (UART_PIN(n, BAUDRATE))
```

Comment:
Perhaps:

```suggestion
#define CONSOLE_BAUD(n)  (UART##n##_BAUDRATE))
```

Reply:
So I think we should be at least able ro reduce this to 2 macros instead of 3, shouldn't we?

Reply:
Yeah, that's gonna work. Pushed an updated commit.